app/controllers/spotlight/roles_controller.rb

```diff
@@ -30,12 +30,39 @@ def update_all
       end
     end
 
+    def exists
+      # note: the messages returned are not shown to users and really only useful for debug, hence no translation necessary
+      #  app uses html status code to act on response
+      if ::User.where(email: exists_params).present?
+        render json: { message: 'User exists' }
+      else
+        render json: { message: 'User does not exist' }, status: :not_found
+      end
+    end
+
+    def invite
+      params.require(:user)
```

Comment:
Should these requires be moved into e.g. `#invite_params`, and have it return e.g. `params.permit(:user, :role)` to match the other methods?